### ISSUE TITLE
only privacy policy text from site 

### ISSUE THREAD

**Issue by fugo** [krzysztofgal/GravGdprPrivacySetupPlugin#11]
How I can use only a small bunch of text in the modal?
the plugin renders always the whole privacy policy page.
In the tab **content** of the plugin I can put whatever I want it will not be desplayed.
In the tab **CSP** -> `(Optional) Url or CSP value` I put nothing.

Whats wrong?

**Comment by krzysztofgal** [comment]
**Url to site policy page** (if site not exists then Privacy Information Field will be used). You have pointed plugin to display Your other privacy information page so plugin uses it instead of content from plugin settings. Change **Url to site policy page** to **not** existing page.

**Comment by fugo** [comment]
this I tried but it's not working:
1. Plugin tab: URL to not existing page + CSP tab: URL to not existing page → `Request failed. Return status of 404`
1. Plugin tab: URL to not existing page +CSP tab: empty → `Request failed. Return status of 404`
1. Plugin tab: URL to empty existing page + CSP tab: URL to not existing page → **empty page** Plugin tab
1. Plugin tab: URL to not existing page + CSP tab: URL to existing page → `Request failed. Return status of 404`
1. Plugin tab: URL to not existing page + CSP tab: `'default-src ''self'' data: ''unsafe-inline'' ''unsafe-eval'''`→ `Request failed. Return status of 404`
1. Plugin tab: `'default-src ''self'' data: ''unsafe-inline'' ''unsafe-eval'''` + CSP tab: `'default-src ''self'' data: ''unsafe-inline'' ''unsafe-eval'''`→ `Request failed. Return status of 404`
**Never** I got the content from Content tab from text field *Privacy Information*

I give up and use now the plugin in **Coolue Consent** it works- Sorry.
